asyncpg/connect_utils.py

```diff
@@ -167,7 +166,9 @@ def _read_password_from_pgpass(
     return None
 
 
-def _validate_port_spec(hosts, port):
+def _validate_port_spec(
+    hosts: Sequence[object], port: typing.Union[int, typing.List[int]]
```

Comment:
```suggestion
    hosts: typing.List[str], port: typing.Union[int, typing.List[int]]
```

Reply:
I did this because of line 172. If your suggestion is correct we can remove the `else` branch. Since I didn't want to change the functionality of the code too much I just added what the code is able to handle instead of what it likely should be, if that makes sense.

Would you want me to change this? Or is keeping as is fine?